tests/WebTestCase.php

```diff
@@ -210,4 +212,13 @@ public function asReviewer(int $id = 1): self
 
         return $this;
     }
+
+    public function passCsrfValidator()
```

Comment:
Can we add a return type declaration as well?

Other than this, this entire thing looks great to me, @BackEndTea!